qcodes/instrument_drivers/Keysight/keysight_e4980a.py

```diff
@@ -334,6 +334,17 @@ def __init__(self,
                       "set."
         )
 
+        self.add_parameter(
+            "CPG_meas",
+            get_cmd=self._get_data,
+            names=("capacitance", "conductance"),
+            units=("F", "S"),
+            parameter_class=MultiParameter,
```

Comment:
I don't think this will work. I would subclass MultiParameter and add the _get_data as a get_raw method